app-text/mdp/mdp-1.0.15.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A markdown presentation tool"
+DESCRIPTION="A command-line based markdown presentation tool"
+HOMEPAGE="https://github.com/visit1985/mdp"
+COPYRIGHT="2018 Michael Goehler and others"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/visit1985/mdp/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="3edc8ea1551fdf290d6bba721105e2e2c23964070ac18c13b4b8d959cdf6116f"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	mdp$secondaryArchSuffix = $portVersion
+	cmd:mdp$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
```

Comment:
Isn't it libncursesw?

Reply:
i have pushed an update that uses `ncursesw` rather than `ncurses`. thanks!

Reply:
REQUIRES still pending.

Reply:
i pushed an update to handle the REQUIRES. i had updated the BUILD_REQUIRES and somehow missed the REQUIRES, thanks for catching that!